### ISSUE TITLE
Cmdlet to Remove or Uninstall a PackageProvider

### ISSUE THREAD

**Issue by damnhandy** [OneGet/oneget#165]
Registering a PackageProvider is pretty straight forward via the Import-PackageProvider and Install-PackageProvider cmdlets. What is not so clear is how to remove a package provider. I am trying my hand at creating a provider but I can't seem to remove the provider. I have removed the module and removed all traces of the module from the `$PSModulePath` but `Get-PackageProvider -listavailable` still shows the provider as present. Are there any plans to have a `Remove-PackageProvider` cmdlet?

**Comment by apetitjean** [comment]
Indeed this is a missing cmdlet. I also would like to have it because as we have ```Install-PackageProvider``` we should have ```Uninstall-PackageProvider```.

**Comment by orangudan** [comment]
Also agree with this.

**Comment by TwoSeven99** [comment]
I think it is pretty awful having an install but no uninstall.

**Comment by blackjack4494** [comment]
Yup. And I still have no clue on how to remove a PackageProvider

**Comment by SjoerdV** [comment]
Pretty obvious oversight here, me thinks...

**Comment by sjortiz** [comment]
ok, but what this ever addressed ?

**Comment by jimbo8098** [comment]
It doesn't seem so no, I still am not able to find an Uninstall-PackageProvider cmdlet
